spec/firebase_test_runner_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+DEFAULT_FILE = '/etc/hosts'.freeze
+RUNNER_TEMP_FILE = Tempfile.new(%w[output log]).path
+
+describe Fastlane::FirebaseTestRunner do
+  describe 'initialize' do
+    it 'runs the correct command' do
+      expect(Fastlane::Action).to receive('sh').with('gcloud', 'auth', 'activate-service-account', '--key-file', __FILE__)
+      described_class.new(key_file: __FILE__)
+    end
```

Comment:
⚠️  We should also add a test to ensure that the authentication is NOT performed if `automatic_login` is `false` and not the default `true` value.

(Though if you apply my earlier suggestion of making the auto-login being performed only during first call to `run_tests` rather than during `new` to avoid side-effects-on-initialize, this current test and the new one covering `auto_login: false` would move elsewhere in the test suite of course)

Reply:
Like you say, this is no longer needed 🙂